deploy/weechat.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+# Simple script to deploy certificates for Weechat relay servers
+# 
+# Configuration:
+# export WEECHAT_PEM (or set in access.conf) to the PEM file you have your weechat client
+# set to load.
+# Optionally configure WEECHAT_HOME if you would like to attempt to reload the certificate
+# on a successful deploy.  
+# This deploy script attempts to guess sane defaults in the absence of either
+
+# If you would like this script to automatically reload this certificate, you must ensure
+# weechat is configured with plugins.var.fifo.fifo = on
+
+# Usage Example: acme.sh --renew --deploy --deploy-hook weechat -d weechat.example.com --force
```

Comment:
carried over from testing by mistake, removed.  I have left the --force in the example so it's obvious to a user how to test if it's initially working.